### PR TITLE
chore: bump package.json files (RHIDP-1720) in main to x.(y+1).0 to avoid conflict with 1.1.x branch

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: bump package.json files in main so they're x.(y+1).0 and therefore greater than those in 1.1.x branch ([RHIDP-1720](https://issues.redhat.com//browse/RHIDP-1720))

This will allow us to do semantic-release releases from the 1.yy.x branches as well as from main, making it more clear which plugins are for RHDH 1.1.z vs. vrom 1.2, and allowing plugin vendors to have maintenance releases too. (eg., Orchestrator and Notifications have asked for this)

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://issues.redhat.com//browse/RHIDP-1720

### How to test this PR?
This PR should trigger releases of all the plugins to npmjs.org via the semantic release process so we have versions tha are greater than those in the 1.1.x branch. 

In future this change would be done by script:  https://gitlab.cee.redhat.com/rhidp/rhdh/-/commit/a88232220e9fe2b10a0858aab20f3ceee6cbb92c

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: plugins/feedback-backend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-feedback-backend",
-  "version": "1.1.3",
+  "version": "1.2.0",
```
Comment (invincibleJai) [inline plugins/feedback-backend/package.json:3]: This version is not published yet? and similar could be the case with other packages

https://www.npmjs.com/package/@janus-idp/backstage-plugin-feedback-backend

Comment (nickboldt) [reply plugins/feedback-backend/package.json:3]: yes! that way main branch will be x.(y+1).0 while 1.2.x branch is at x.y.z, and can keep doing .z updates that won't collide with the x.(y+1) stuff in main. And main can rev to x.(y+2) if needed as well, as long as it's got a larger y-digit than the 1.yy.x maintenance branches

**Comment by invincibleJai** [comment]
Thanks @nickboldt,

I liked the approach with this change fixes can cherry picked to 1.1.x and it can do a release independently from the respective branch.

For this PR I have two doubts
- The job is failing https://app.snyk.io/org/janus-idp/pr-checks/804dd0e0-5485-474a-9782-de0208217d89 and couldn't see the failures but this can be due to version not found as we are bumping and in npm that version might not exist. So this PR should be the exception and get manual merge and post merge do we expect the release to trigger for this version bumped with this PR otherwise all open PR would see these failures 
- With the current setup release doesn't happen from 1.1.x if am not wrong so that needs to be supported as well

cc @gashcrumb @kadel

File: packages/app/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "app",
-  "version": "0.0.0",
+  "version": "0.1.0",
```
Comment (nickboldt) [inline packages/app/package.json:3]:
```suggestion
  "version": "0.0.0",
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (nickboldt) [inline package.json:3]:
```suggestion
  "version": "3.2.0",
```

File: packages/storybook/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "storybook",
-  "version": "0.0.0",
+  "version": "0.1.0",
```
Comment (nickboldt) [inline packages/storybook/package.json:3]:
```suggestion
  "version": "0.0.0",
```

File: packages/backend/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backend",
-  "version": "0.0.0",
+  "version": "0.1.0",
```
Comment (nickboldt) [inline packages/backend/package.json:3]:
```suggestion
  "version": "0.0.0",
```

**Comment by nickboldt** [comment]
@schultzp2020 I've reverted the bumps in packages/ to keep them at 0.0.0 if they're currently 0.0.0;
@invincibleJai Here's the PR for enabling 1.yy.x semantic releases https://github.com/janus-idp/backstage-plugins/pull/1401 which we can merge after this one.

**Review by schultzp2020 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**